torch/_inductor/config.py

```diff
@@ -345,6 +345,12 @@ class cpp:
     # using atomic_add.
     fallback_scatter_reduce_sum = True
 
+    # Allow the model to cache and reuse outputs across iterations. REQUIRES that the
+    # calling code guarantee it has stopped using previous iterations' outputs before
+    # calling into the model again. Currently needed to enable reusing internal Tensor
+    # allocations generally before we land proper memory planning.
+    allow_unsafe_output_reuse = False
```

Comment:
TODO: need to make sure we are running as many tests as possible in this configuration and inspect failures, otherwise this diff will effectively be untested